mix.exs

```diff
@@ -173,6 +173,7 @@ defmodule Oli.MixProject do
       {:libcluster_ec2, "~> 0.6"},
       {:mime, "~> 1.2"},
       {:mix_test_watch, "~> 1.0", only: :dev, runtime: false},
+      {:monocle, "~> 0.0.1"},
```

Comment:
Could you please reference this new dependency in the PR description and briefly explain why are we adding it?